sympy/matrices/matrices.py

```diff
@@ -4130,9 +4130,28 @@ def QRsolve(self, b):
     def solve_least_squares(self, rhs, method='CH'):
         """Return the least-square fit to the data.
 
-        By default the cholesky_solve routine is used (method='CH'); other
-        methods of matrix inversion can be used. To find out which are
-        available, see the docstring of the .inv() method.
+        Parameters
+        ==========
+
+        rhs : Matrix
+            Vector representing the right hand side of the linear equation.
+
+        method : string or boolean, optional
+            If set to ``'CH'``, ``cholesky_solve`` routine will be used.
+
+            If set to ``'QR'``, ``QRsolve`` routine will be used.
+
+            If set to ``'PINV'``, ``pinv_solve`` routine will be used.
+
+            Other linear solving methods may be used, which can be inherited
```

Comment:
This is a little unlcear to me. Maybe this edit?

For any other method, a naive approach will be taken using the conjugate of self; the passed method will be passed to `solve` method.